src/dataworker/DataworkerUtils.ts

```diff
@@ -60,10 +60,14 @@ export async function blockRangesAreInvalidForSpokeClients(
     // There should be a spoke pool client instantiated for every bundle timestamp.
     assert(!Object.keys(endBlockTimestamps).some((chainId) => !isDefined(spokePoolClients[chainId])));
   }
-  return utils.someAsync(blockRanges, async ([start, end], index) => {
+  let reason: string | undefined;
+  const _result = await utils.someAsync(blockRanges, async ([start, end], index) => {
```

Comment:
ooc, why is this `_result` instead of `result` ?

```suggestion
  const result = await utils.someAsync(blockRanges, async ([start, end], index) => {
```

Reply:
770738fabd6c76bb2ff259a680a5450967d19674